tests/sieves/_pulp/test_no_index_label.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+# thoth-adviser
+# Copyright(C) 2019 - 2021 Fridolin Pokorny
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Test a sieve that provides only packages available on Operate First Pulp instance."""
```

Comment:
```suggestion
"""Test a sieve that provides only packages available on PyPI instance."""
```

Reply:
This is specific to Pulp instance (an alternative to PyPI that is being provisioned). See https://www.operate-first.cloud/community-handbook/pulp/usage.md

Reply:
Ok I see, thanks!